.gitignore

```diff
@@ -11,7 +11,6 @@ dist/
 
 # Dependency directories
 node_modules
-jspm_packages
```

Comment:
We don't have a jspm_packages folder. This was there just because I had copy/pasted from our codebase.